client/views/actions/kick.tpl

```diff
@@ -1,6 +1,6 @@
 {{> ../user_name nick=from}}
 has kicked
-{{> ../user_name nick=target}}
+{{> ../user_name nick=target mode=""}}
```

Comment:
Can kick target be voiced or op or whatever? In other word, why would it always be empty mode?